File: core/src/core/java/nativefont/NativeFontListener.java
package core.java.nativefont;

import com.badlogic.gdx.graphics.Pixmap;

/**
 * Created by tian on 2016/10/2.
 */

public interface NativeFontListener {
    Pixmap getFontPixmap(String str, NativeFontPaint freePaint);
}

File: core/src/core/java/nativefont/NativeFontListener.kt
package core.java.nativefont

import com.badlogic.gdx.graphics.Pixmap

/**
 * Created by tian on 2016/10/2.
 */
interface NativeFontListener {
    fun getFontPixmap(txt: String, vpaint: NativeFontPaint): Pixmap
}